tutorial/Lab2/README.md

```diff
@@ -1,25 +1,29 @@
 # Lab 2. I need to change but want none of the hassle
 
-In [Lab 1](../Lab1/README.md), we installed the guestbook sample app by using Helm and saw the benefits over using `kubectl`. You probably think that you're done and know enough to use Helm. But what about updates or improvements to the chart? How do you update your running app to pick up these changes? 
+In [Lab 1](../Lab1/README.md), we installed the guestbook sample app by using Helm and saw the benefits over using `kubectl`. You probably think that you're done and know enough to use Helm. But what about updates or improvements to the chart? How do you update your running app to pick up these changes?
 
 In this lab, we're going to look at how to update our running app when the chart has been changed. To demonstrate this, we're going to make changes to the original `guestbook` chart by:
+
 * Removing the Redis slaves and using just the in-memory DB
 * Changing the type from `LoadBalancer` to `NodePort`.
 
 It seems contrived but the goal of this lab is to show you how to update your apps with Kubernetes and Helm. So, how easy is it to do this? Let's take a look below.
 
-# Update the application using `kubectl`
+## Scenario 1: Update the application using `kubectl`
 
 In this part of the lab we will update the previously deployed application [Guestbook](https://github.com/IBM/guestbook), using Kubernetes directly.
 
-1. This is an optional step, that it is not necessary to update your running app. The reason for doing this step is "house keeping" - you want to have the correct files for the current configuration that you have deployed. This avoids making mistakes if you have future updates or even rollbacks. In this updated configuration, we remove the Redis slaves. That is why you should move/archive or remove the Redis slave files to keep abreast of the update:
+1. This is an **optional** step that is not technically required to update your running app. The reason for doing this step is "house keeping" - you want to have the correct files for the current configuration that you have deployed. This avoids making mistakes if you have future updates or even rollbacks. In this updated configuration, we remove the Redis slaves. To have the directory match the configuration, move/archive or simply remove the Redis slave files from the guestbook repo tree:
 
+   ``` console
+   cd guestbook/v1
+   rm redis-slave-service.yaml
+   rm redis-slave-deployment.yaml
    ```
-   redis-slave-service.yaml
-   redis-slave-deployment.yaml
-   ```
 
-2. Delete the Redis slave service and pods:
+   > Note: you can reclaim these files later with a `git checkout -- <filemane>` command, if desired
```

Comment:
I think the example is correct. Without the `--` the checkout command argument is presumed to be a branch rather than a filespec. See: https://git-scm.com/docs/git-checkout

Reply:
yeah, good point. Both versions will work but better to follow the standard.

Reply:
filemane --> filename

Reply:
d'oh e2f9fd4